charts/clickhouse/templates/clickhouse-operator/configmaps/etc-files.yaml

```diff
@@ -0,0 +1,288 @@
+          # respectfully
+          wait:
+            exclude: true
+            include: false
+
+      ################################################
+      ##
+      ## Annotations management
+      ##
+      ################################################
+      annotation:
+        # Applied when:
+        #  1. Propagating annotations from the CHI's `metadata.annotations` to child objects' `metadata.annotations`,
+        #  2. Propagating annotations from the CHI Template's `metadata.annotations` to CHI's `metadata.annotations`,
+        # Include annotations from the following list:
+        # Applied only when not empty. Empty list means "include all, no selection"
+        include: []
+        # Exclude annotations from the following list:
+        exclude: []
+
+      ################################################
+      ##
+      ## Labels management
+      ##
+      ################################################
+      label:
+        # Applied when:
+        #  1. Propagating labels from the CHI's `metadata.labels` to child objects' `metadata.labels`,
+        #  2. Propagating labels from the CHI Template's `metadata.labels` to CHI's `metadata.labels`,
+        # Include labels from the following list:
+        # Applied only when not empty. Empty list means "include all, no selection"
+        include: []
+        # Exclude labels from the following list:
+        # Applied only when not empty. Empty list means "nothing to exclude, no selection"
+        exclude: []
+        # Whether to append *Scope* labels to StatefulSet and Pod.
+        # Full list of available *scope* labels check in 'labeler.go'
+        #  LabelShardScopeIndex
+        #  LabelReplicaScopeIndex
+        #  LabelCHIScopeIndex
+        #  LabelCHIScopeCycleSize
+        #  LabelCHIScopeCycleIndex
+        #  LabelCHIScopeCycleOffset
+        #  LabelClusterScopeIndex
+        #  LabelClusterScopeCycleSize
+        #  LabelClusterScopeCycleIndex
+        #  LabelClusterScopeCycleOffset
+        appendScope: "no"
+
+      ################################################
+      ##
+      ## StatefulSet management
+      ##
+      ################################################
+      statefulSet:
+        revisionHistoryLimit: 0
+
+      ################################################
+      ##
+      ## Pod management
+      ##
+      ################################################
+      pod:
+        # Grace period for Pod termination.
+        # How many seconds to wait between sending
+        # SIGTERM and SIGKILL during Pod termination process.
+        # Increase this number is case of slow shutdown.
+        terminationGracePeriod: 30
+
+      ################################################
+      ##
+      ## Log parameters
+      ##
+      ################################################
+      logger:
+        logtostderr: "true"
+        alsologtostderr: "false"
+        v: "1"
+        stderrthreshold: ""
```

Comment:
I am assuming the single `configmap.yaml` is split into multiple files under `configmaps` without any changes (except ttl templating).

Reply:
yes, you are right.